gilded-rose-llc-frontend/package.json

```diff
@@ -15,7 +15,7 @@
     "react-dom": "18.2.0"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
Question: Anything difference from the previous version?

Reply:
Honestly, I have no idea why that changed.

Is there something I did to cause this update? Maybe when I ran `yarn install` after recloning the repo it updated to the most recent version?

This brings up a question: **Is it best practice to `.gitignore` the lock files? It looks like the consensus is "it depends" where keeping the lock files helps with version control (probably avoiding this from happening)**

Reply:
`yarn install` should not change the `package.json` file. IMO `yarn.lock` **should** be tracked in version control.

Reply:
https://yarnpkg.com/getting-started/qa/#should-lockfiles-be-committed-to-the-repository

Reply:
Perhaps you might have run yarn install @testing-library/jest-dom after cloning the repo without knowing that it's already installed?

In general, you want to commit the lock file changes to upstream. More detail [here](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/)

Reply:
Ok, thanks for the resources here. I added the lock file back in here [022a365](https://github.com/jramnani/GildedRoseJavaLLC/pull/32/commits/022a365c94237494fc8a40327ef134351c1c0d6a)